### ISSUE TITLE
macOS (via home-manager) somewhere out there?

### ISSUE THREAD

**Issue by AngryAnt** [nixified-ai/flake#30]
Some of our use cases are macOS based and our flow would benefit greatly from bringing those M-class cores to good use on AI crunch. I realize that those are two separate challenges, but was wondering if you could possibly provide a broad idea on what feasibility and vague schedule range on something like that would look like. "Never going to work" / "Not this year" / "mac yes, M-chip, no" etc.

**Comment by MatthewCroughan** [comment]
Yes to all, I just need money and hardware. I don't own any M-chip hardware, x86_64-darwin hardware, etc. I can maintain these things if I were to be sent hardware, and potentially rewarded financially in some way. There isn't much incentive to support macOS otherwise.

**Comment by MatthewCroughan** [comment]
> We're a two-person shop hacking at this

More than happy to accept contributions and add you to the AUTHORs file, and also send funds your way whenever they roll in. I'm waiting on GitHub sponsors in order to start receiving funds for the project. Microsoft are taking their sweet time to approve that, maybe they watched the video and decided to shadow ban me.
